package.json

```diff
@@ -155,6 +155,12 @@
                     "type": "number",
                     "default": 5000,
                     "description": "%AWS.ssmDocument.ssm.maxItemsComputed.desc%"
+                },
+                "aws.cloudWatchLogs.limit": {
+                    "type": "number",
+                    "default": 1000,
```

Comment:
Do we not want the default to be 10000? That's what we do in JB and as a user I like it more since it means less page loads

Reply:
Adjusted back